engine/tester/config_parse.c

```diff
@@ -1009,13 +1151,22 @@ get_test_attrs(xmlNodePtr node, test_attrs *attrs)
 static te_errno
 get_script(xmlNodePtr node, tester_cfg *cfg, test_script *script)
 {
-    te_errno            rc;
-    const test_info    *ti;
-    bool                objective_found = false;
-    bool                execute_found = false;
+    te_errno         rc;
+    const test_info *ti;
+    bool             objective_found = false;
+    bool             execute_found = false;
```

Comment:
Let's refrain from re-aligning variable blocks: our current Coding Standard explicitly discourages this.